projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-ai.php

```diff
@@ -28,14 +28,11 @@ class WPCOM_REST_API_V2_Endpoint_AI extends WP_REST_Controller {
 	 * WPCOM_REST_API_V2_Endpoint_AI constructor.
 	 */
 	public function __construct() {
-		$this->is_wpcom = false;
+		$this->is_wpcom                     = false;
```

Comment:
Do we actually use/need this?

Reply:
I checked and it does not seem to be used anywhere...

CC @artpi

Reply:
I think when @jeherve introduced this he meant to use `wpcom_is_wpcom_only_endpoint`. I think we could remove it